packages/eslint-config-airbnb/rules/variables.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     // disallow shadowing of names such as arguments
     'no-shadow-restricted-names': 2,
     // disallow declaration of variables already declared in the outer scope
-    'no-shadow': 2,
+    'no-shadow': [2, {'builtinGlobals': true}],
```

Comment:
please flesh out all the config options here - ie, `hoist`, and `allow`, with their defaults.